explorer/src/components/Domain/DomainBlockTime.tsx

```diff
@@ -103,7 +110,21 @@ export const DomainBlockTimeProgress: FC<DomainBlockTimeProgressProps> = ({
           <div key={statIndex} className='mb-3 sm:mb-4'>
             <div className='flex items-center justify-between'>
               <span className='text-base font-semibold text-grayDark dark:text-white sm:text-lg'>
-                {title}
+                <Tooltip
```

Comment:
<img width="782" alt="image" src="https://github.com/user-attachments/assets/d11e5497-3dbb-46b6-b027-294d12e06d63">
Tooltip doesn't overflow the table and gets cut

Reply:
Thanks, that is addressed by setting the direction of the tooltip